javascript-practice/src/pages/dashboard.html

```diff
@@ -23,16 +22,16 @@ <h2 class="header-dashboard-name">User</h2>
           <div class="header-dashboard-profile-box flex">
             <div class="header-dashboard-avatar"></div>
             <div>
-              <!-- TODO: this is fake data. I'll update it in the next task -->
               <div class="flex header-dashboard-profile-name-box">
-                <h2 class="header-dashboard-profile-name">FirstName</h2>
-                <h2 class="header-dashboard-profile-name">LastName</h2>
+                <h2 id="firstName" class="header-dashboard-profile-name"></h2>
```

Comment:
I think you just need only one element `h2` for both the First and Last Name. And we will format the name in JS and append it here

Reply:
i done